sklearn/metrics/ranking.py

```diff
@@ -328,7 +328,7 @@ def _binary_roc_auc_score(y_true, y_score, sample_weight=None):
         if max_fpr is None or max_fpr == 1:
             return auc(fpr, tpr)
         if max_fpr <= 0 or max_fpr > 1:
-            raise ValueError("Expected max_frp in range ]0, 1], got: %r"
+            raise ValueError("Expected max_fpr in range [0, 1), got: %r"
```

Comment:
I think that range notation should equate to (0,1], not [0,1)

Reply:
You're right, jnothman. I have fixed it. Thanks.